lib/puma/plugin/systemd.rb

```diff
@@ -51,11 +51,14 @@ def start(launcher)
   end
 
   def status
-    common = "workers: #{running}/#{max_threads} threads, #{pool_capacity} available, #{backlog} backlog"
     if clustered?
-      "Puma #{Puma::Const::VERSION} cluster: #{booted_workers}/#{workers} #{common}"
+      messages = Array.new(workers) do |i|
+        common_message(stats[:worker_status][i][:last_status])
```

Comment:
I didn't look at the actual source, but would it be possible to iterate over `stats[:worker_status]` instead? Untested, but something like:
```suggestion
      messages = stats[:worker_status].values do |worker|
        common_message(worker[:last_status])
```

Reply:
Thank you!
stats[:worker_status] is Array instanse.
So I modified it as follows

```ruby
      messages = stats[:worker_status].map do |worker|
        common_message(worker[:last_status])
      end.join(',')
```